/*package webapp;

import java.util.Scanner;

public class Exception {
	public static void main(String args[])
	{
		int a,b,c;
		Scanner s=new Scanner(System.in);
		System.out.println("Enter a,b values");
		a=s.nextInt();
		b=s.nextInt();
		try
		{
			c=a/b;
			System.out.println("c value is "+c);
		}
		catch(java.lang.ArithmeticException obj)
		{
			System.out.println("The error is "+obj);
		}
		System.out.println("Hello java");
		System.out.println("Exception Handling");
	}
}*/
